### PR TITLE
RHDEVDOCS-2985 Logging: Create RNs for 5.0.3 RHSA-2021:1515 OSE-LOGGING

### PR DESCRIPTION
**Pull request by rolfedh**
- Aligned team: Dev Tools
- For branches: enterprise-4.7, enterprise-4.8
- https://issues.redhat.com/browse/RHDEVDOCS-2985
- Direct link to doc preview: https://deploy-preview-32292--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html#cluster-logging-release-notes-5-0-3
- SME review: Requested from @vimalk78 
- QE review: Requested from @anpingli 
- Peer review: Requested from team
- <Please merge now>

### CODE REVIEW

File: modules/cluster-logging-release-notes-5.0.3.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+* jackson-databind: mishandles the interaction between serialization gadgets and typing, related to com.newrelic.agent.deps.ch.qos.logback.core.db.JNDIConnectionSource (link:https://www.redhat.com/security/data/cve/CVE-2020-36188.html[*CVE-2020-36188*])
+* jackson-databind: mishandles the interaction between serialization gadgets and typing, related to com.newrelic.agent.deps.ch.qos.logback.core.db.DriverManagerConnectionSourc e (link:https://www.redhat.com/security/data/cve/CVE-2020-36189.html[*CVE-2020-36189*])
+* jackson-databind: mishandles the interaction between serialization gadgets and typing, related to javax.swing (link:https://www.redhat.com/security/data/cve/CVE-2021-20190.html[*CVE-2021-20190*])
+* golang: data race in certain net/http servers including ReverseProxy can lead to DoS (link:https://www.redhat.com/security/data/cve/CVE-2020-15586.html[*CVE-2020-15586*])
+* golang: ReadUvarint and ReadVarint can read an unlimited number of bytes from invalid inputs (link:https://www.redhat.com/security/data/cve/CVE-2020-16845.html[*CVE-2020-16845*])
+* (link:https://www.redhat.com/security/data/cve/CVE-2021-2163.html[*CVE-2021-2163*])
```
Comment (Srivaralakshmi) [inline modules/cluster-logging-release-notes-5.0.3.adoc:35]: I see that [this page](https://access.redhat.com/security/cve/CVE-2021-2163) has the description inside the Additional Information box, as: OpenJDK: Incomplete enforcement of JAR signing disabled algorithms (Libraries, 8249906)

Suggest to add the preceding description if it is appropriate.

File: modules/cluster-logging-release-notes-5.0.3.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The following Jira issues contain the above CVE's:
+
+* LOG-1234 CVE-2020-15586 CVE-2020-16845 openshift-eventrouter: various flaws [openshift-4]. (link:https://issues.redhat.com/browse/LOG-1234[*LOG-1234*])
```
Comment (Srivaralakshmi) [inline modules/cluster-logging-release-notes-5.0.3.adoc:39]: Suggest: If possible, try requesting for a change in the title that makes it meaningful. It does not look good nor is it readable.

File: modules/cluster-logging-release-notes-5.0.3.adoc [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+* LOG-1243 CVE-2018-14718 CVE-2018-14719 CVE-2018-14720 CVE-2018-14721 CVE-2018-19360 CVE-2018-19361 CVE-2018-19362 CVE-2019-14379 CVE-2020-35490 CVE-2020-35491 CVE-2020-35728... logging-elasticsearch6-container: various flaws [openshift-logging-5.0]. (link:https://issues.redhat.com/browse/LOG-1243[*LOG-1243*])
```
Comment (Srivaralakshmi) [inline modules/cluster-logging-release-notes-5.0.3.adoc:40]: Suggest: If possible, try requesting for a change in the title that makes it meaningful. It does not look good nor is it readable.

**Review by Srivaralakshmi — COMMENTED** [review]
Hi @rolfedh Thanks for the peer-review opportunity. I have left some comments. PTAL.

File: modules/cluster-logging-release-notes-5.0.3.adoc [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+* jackson-databind: mishandles the interaction between serialization gadgets and typing, related to com.newrelic.agent.deps.ch.qos.logback.core.db.DriverManagerConnectionSourc e (link:https://www.redhat.com/security/data/cve/CVE-2020-36189.html[*CVE-2020-36189*])
+* jackson-databind: mishandles the interaction between serialization gadgets and typing, related to javax.swing (link:https://www.redhat.com/security/data/cve/CVE-2021-20190.html[*CVE-2021-20190*])
+* golang: data race in certain net/http servers including ReverseProxy can lead to DoS (link:https://www.redhat.com/security/data/cve/CVE-2020-15586.html[*CVE-2020-15586*])
+* golang: ReadUvarint and ReadVarint can read an unlimited number of bytes from invalid inputs (link:https://www.redhat.com/security/data/cve/CVE-2020-16845.html[*CVE-2020-16845*])
+* (link:https://www.redhat.com/security/data/cve/CVE-2021-2163.html[*CVE-2021-2163*])
+
+The following Jira issues contain the above CVE's:
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:37]: Thanks. Updated.

File: logging/cluster-logging-release-notes.adoc [hunk 5]
```diff
@@ -13,17 +13,13 @@ The following advisories are available for {ProductName} 5.0:
 * link:https://access.redhat.com/errata/RHBA-2021:0652[RHBA-2021:0652 Errata Advisory for Openshift Logging 5.0.0]
 * link:https://access.redhat.com/errata/RHBA-2021:0963[RHBA-2021:0963 for OpenShift Logging Bug Fix Release (5.0.1)]
 * link:https://access.redhat.com/errata/RHBA-2021:1167[RHBA-2021:1167 for OpenShift Logging Bug Fix Release (5.0.2)]
+* link:https://access.redhat.com/errata/RHSA-2021:1515[RHSA-2021:1515 - Security Advisory. Important: Openshift Logging Bug Fix Release (5.0.3)]
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/cluster-logging-release-notes.adoc:16]: Thanks, updated the first issue. Unfortunately, I can't edit or update the advisory Authored by the Security team.

File: modules/cluster-logging-release-notes-5.0.3.adoc [hunk 6]
```diff
@@ -0,0 +1,49 @@
+[id="cluster-logging-release-notes-5-0-3"]
+= OpenShift Logging 5.0.3
+
+This release includes Red Hat Bug Advisory, link:https://access.redhat.com/errata/RHSA-2021:1515[RHSA-2021:1515 - Security Advisory. Important: OpenShift Logging Bug Fix Release (5.0.3)].
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:4]: Thanks. Updated.

File: modules/cluster-logging-release-notes-5.0.3.adoc [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+[id="openshift-logging-5-0-3-security-fixes"]
+== Security fixes
+
+* jackson-databind: arbitrary code execution in slf4j-ext class (link:https://www.redhat.com/security/data/cve/CVE-2018-14718.html[*CVE-2018-14718*])
+* jackson-databind: arbitrary code execution in blaze-ds-opt and blaze-ds-core classes (link:https://www.redhat.com/security/data/cve/CVE-2018-14719.html[*CVE-2018-14719*])
+* jackson-databind: exfiltration/XXE in some JDK classes (link:https://www.redhat.com/security/data/cve/CVE-2018-14720.html[*CVE-2018-14720*])
+* jackson-databind: server-side request forgery (SSRF) in axis2-jaxws class (link:https://www.redhat.com/security/data/cve/CVE-2018-14721.html[*CVE-2018-14721*])
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:12]: Sorry. It doesn't seem to be a proper noun, just an ordinary noun that has an initialism.

[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:12]: Also, I can't edit the CVEs and don't want to create mismatches between CVEs and the release notes.

File: modules/cluster-logging-release-notes-5.0.3.adoc [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+* jackson-databind: improper polymorphic deserialization in axis2-transport-jms class (link:https://www.redhat.com/security/data/cve/CVE-2018-19360.html[*CVE-2018-19360*])
+* jackson-databind: improper polymorphic deserialization in openjpa class (link:https://www.redhat.com/security/data/cve/CVE-2018-19361.html[*CVE-2018-19361*])
+* jackson-databind: improper polymorphic deserialization in jboss-common-core class (link:https://www.redhat.com/security/data/cve/CVE-2018-19362.html[*CVE-2018-19362*])
+* jackson-databind: default typing mishandling leading to remote code execution (link:tbhttps://www.redhat.com/security/data/cve/CVE-2019-14379.htmld[*CVE-2019-14379*])
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:16]: Nice catch! Thanks. Updated.

File: modules/cluster-logging-release-notes-5.0.3.adoc [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+* jackson-databind: Serialization gadgets in com.pastdev.httpcomponents.configuration.JndiConfiguration (link:https://www.redhat.com/security/data/cve/CVE-2020-24750.html[*CVE-2020-24750*])
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:17]: Updated. Thanks.

File: modules/cluster-logging-release-notes-5.0.3.adoc [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+* jackson-databind: mishandles the interaction between serialization gadgets and typing, related to org.apache.tomcat.dbcp.dbcp.datasources.PerUserPoolDataSource (link:https://www.redhat.com/security/data/cve/CVE-2020-36186.html[*CVE-2020-36186*])
+* jackson-databind: mishandles the interaction between serialization gadgets and typing, related to org.apache.tomcat.dbcp.dbcp.datasources.SharedPoolDataSource (link:https://www.redhat.com/security/data/cve/CVE-2020-36187.html[*CVE-2020-36187*])
+* jackson-databind: mishandles the interaction between serialization gadgets and typing, related to com.newrelic.agent.deps.ch.qos.logback.core.db.JNDIConnectionSource (link:https://www.redhat.com/security/data/cve/CVE-2020-36188.html[*CVE-2020-36188*])
+* jackson-databind: mishandles the interaction between serialization gadgets and typing, related to com.newrelic.agent.deps.ch.qos.logback.core.db.DriverManagerConnectionSourc e (link:https://www.redhat.com/security/data/cve/CVE-2020-36189.html[*CVE-2020-36189*])
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:31]: Updated. Thanks.

File: modules/cluster-logging-release-notes-5.0.3.adoc (hunk 1 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:35]: Thank you. This information was not available earlier and is still offline now. Updated.

File: modules/cluster-logging-release-notes-5.0.3.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:37]: Updated. Thanks.

File: logging/cluster-logging-release-notes.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/cluster-logging-release-notes.adoc:16]: Updated. Thanks.

File: modules/cluster-logging-release-notes-5.0.3.adoc (hunk 6 above)
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:4]: Updated. Thanks.

File: modules/cluster-logging-release-notes-5.0.3.adoc (hunk 2 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:39]: I see your point, but I don't think it is appropriate to request that engineering and security make these revisions now.

File: modules/cluster-logging-release-notes-5.0.3.adoc (hunk 3 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:40]: I see your point, but I don't think it is appropriate to request that engineering and security make these revisions now.

**Comment by rolfedh** [comment]
Thank you for your thorough review and suggestions, @Srivaralakshmi. I have updated the content accordingly.

File: logging/cluster-logging-release-notes.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply logging/cluster-logging-release-notes.adoc]: > Thanks, updated the first issue. Unfortunately, I can't edit or update the advisory authored by the Security team.

@rolfedh ok, I understand. If it is possible, please notify them about this and the need to rectify it.

File: modules/cluster-logging-release-notes-5.0.3.adoc (hunk 7 above)
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply modules/cluster-logging-release-notes-5.0.3.adoc:13]: > Sorry. It doesn't seem to be a proper noun, just an ordinary noun that has an initialism.

Ack. :)

**Review by Srivaralakshmi — COMMENTED** [review]
@rolfedh Done with review.. just some comments.. PTAL. Thanks!

File: logging/cluster-logging-release-notes.adoc [hunk 11]
```diff
@@ -10,20 +10,16 @@ toc::[]
 
 The following advisories are available for {ProductName} 5.0:
 
-* link:https://access.redhat.com/errata/RHBA-2021:0652[RHBA-2021:0652 Errata Advisory for Openshift Logging 5.0.0]
-* link:https://access.redhat.com/errata/RHBA-2021:0963[RHBA-2021:0963 for OpenShift Logging Bug Fix Release (5.0.1)]
-* link:https://access.redhat.com/errata/RHBA-2021:1167[RHBA-2021:1167 for OpenShift Logging Bug Fix Release (5.0.2)]
+* link:https://access.redhat.com/errata/RHBA-2021:0652[RHBA-2021:0652 Bug Fix Advisory for OpenShift Logging 5.0.0]
+* link:https://access.redhat.com/errata/RHBA-2021:0963[RHBA-2021:0963 Bug Fix Advisory for OpenShift Logging Bug Fix Release (5.0.1)]
+* link:https://access.redhat.com/errata/RHBA-2021:1167[RHBA-2021:1167 Bug Fix Advisory for OpenShift Logging Bug Fix Release (5.0.2)]
+* link:https://access.redhat.com/errata/RHSA-2021:1515[RHSA-2021:1515 Security Advisory for Important OpenShift Logging Bug Fix Release (5.0.3)]
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply logging/cluster-logging-release-notes.adoc:16]: I'm re-using the language in the Security Advisory. If they say it's important, I don't want to omit that from the description.

File: modules/cluster-logging-release-notes-5.0.3.adoc [hunk 12]
```diff
@@ -0,0 +1,50 @@
+[id="cluster-logging-release-notes-5-0-3"]
+= OpenShift Logging 5.0.3
+
+This release includes link:https://access.redhat.com/errata/RHSA-2021:1515[RHSA-2021:1515 Security Advisory for Important OpenShift Logging Bug Fix Release (5.0.3)]
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc:4]: Same here.

File: modules/cluster-logging-release-notes-5.0.3.adoc [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+* LOG-1243 CVE-2018-14718 CVE-2018-14719 CVE-2018-14720 CVE-2018-14721 CVE-2018-19360 CVE-2018-19361 CVE-2018-19362 CVE-2019-14379 CVE-2020-35490 CVE-2020-35491 CVE-2020-35728... logging-elasticsearch6-container: various flaws [openshift-logging-5.0]. (link:https://issues.redhat.com/browse/LOG-1243[*LOG-1243*])
+
+[id="openshift-logging-5-0-3-bug-fixes"]
+== Bug fixes
+
+This release also includes the following bug fixes:
+
+* LOG-1224 Release 5.0 - ClusterLogForwarder namespace-specific log forwarding does not work as expected. (link:https://issues.redhat.com/browse/LOG-1224[*LOG-1224*])
+* LOG-1232 5.0 - Bug 1859004 - Sometimes the eventrouter couldn't gather event logs. (link:https://issues.redhat.com/browse/LOG-1232[*LOG-1232*])
+* LOG-1299 Release 5.0 Forwarding logs to Kafka using Chained certificates fails with error "state=error: certificate verify failed (unable to get local issuer certificate)". (link:https://issues.redhat.com/browse/LOG-1299[*LOG-1299*])
```
Comment (Srivaralakshmi) [inline modules/cluster-logging-release-notes-5.0.3.adoc:50]: @rolfedh 

LOG-1299 Release **5.0 Forwarding**

Suggestion: Need a hyphen after 5.0 (needs to be consistent with line 48)

File: modules/cluster-logging-release-notes-5.0.3.adoc (hunk 12 above)
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply modules/cluster-logging-release-notes-5.0.3.adoc:4]: ok.

File: modules/cluster-logging-release-notes-5.0.3.adoc (hunk 2 above)
Comment (Srivaralakshmi) [reply modules/cluster-logging-release-notes-5.0.3.adoc:40]: ok.

File: modules/cluster-logging-release-notes-5.0.3.adoc (hunk 13 above)
Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.0.3.adoc]: Thank you, @Srivaralakshmi. Updated.

**Review by anpingli — COMMENTED** [review]
/lgtm

**Pull request merged**